src/AzureFunctions/deploy.msbuild

```diff
@@ -0,0 +1,94 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Deploy" ToolsVersion="4.0">
+	<PropertyGroup>
+		<GXDeployFileProject>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\..\..\web\$(ProjectName).gxdproj'))</GXDeployFileProject>
+	</PropertyGroup>
+	
+	<Import Project="$(GX_PROGRAM_DIR)\GeneXus.AzureDeploy.targets"/>
+	<Import Project="$(GXDeployFileProject)"/>
+	
+	<PropertyGroup Condition="'$(GENERATOR)' == '.NET Core'">
+		<UploadSource>$(DeployFullPath)</UploadSource>
+		<DeploymentTargetPath>$(GX_PROGRAM_DIR)\DeploymentTargets\AzureFunctions\AzFunctionFiles</DeploymentTargetPath>
+	</PropertyGroup>
+	
+	<ItemGroup>
+		<AllObjects Include="$(ObjectNames.ToLower().Replace(`procedure:`, ``).Split(`;`))"/>
```

Comment:
This ItemGroup seems to be unused?

Reply:
I started thinking on supporting more than one main object. Now we cannot support it because each should have different settings.

Reply:
I removed the lines.